vcluster/deploy/topologies/high-availability.mdx

```diff
@@ -1,14 +1,16 @@
 ---
 title: Deploy in High Availability
-sidebar_label: High Availability 
+sidebar_label: High Availability
 sidebar_position: 1
 ---
 
-By default, vCluster runs one instance of each of its components. That’s fine for many use cases, like ones that are very ephemeral (dev environments, CI/CD, etc.). But suppose your situation requires virtual clusters with more redundancy. In that case, you can use vCluster’s High Availability feature to run multiple copies of the vCluster components so that the cluster is more resistant to partial failures.
+By default, vCluster runs one instance of each of its components. That’s fine for many use cases, like ones that are very ephemeral (dev environments, CI/CD, etc.).
+Suppose your situation requires virtual clusters with more redundancy. In that case, you can use vCluster’s High Availability (HA) feature
+to run multiple copies of the vCluster components so that the cluster is more resistant to partial failures.
 
 ## Create the HA virtual cluster
 
-To create a virtual cluster using the vcluster CLI, we run the vcluster create command. To enable HA, we’ll need to specify the distro and the values.yaml file to use:
+To create a virtual cluster using the vcluster CLI, we run the `vcluster create` command. To enable HA, we’ll need to specify the distro and the vcluster.yaml file to use:
```

Comment:
Put `vcluster.yaml` in a code formatting in this part as well as any other reference in the doc.